### DISCUSSION TITLE
Add Rio to list of terminal emulators that support the kitty keyboard protocol

### DISCUSSION THREAD

**Discussion by kirawi** [kovidgoyal/kitty#6934] [Ideas]
This was added in https://github.com/raphamorim/rio/commit/cd463ca37677a0fc48daa8795ea46dadc92b1e95

**Comment by kovidgoyal** [comment]
Sure.

On Wed, Dec 20, 2023 at 01:33:06PM -0800, Kirawi wrote:
> This was added in https://github.com/raphamorim/rio/commit/cd463ca37677a0fc48daa8795ea46dadc92b1e95
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/6934
> You are receiving this because you are subscribed to this thread.
> 
> Message ID: ***@***.***>

-- 
_____________________________________

Dr. Kovid Goyal 
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________
